make-util.js

```diff
@@ -393,6 +393,49 @@ var downloadFile = function (url) {
 }
 exports.downloadFile = downloadFile;
 
+// Check if a specified version of a task exists in packaging feed.
+var taskVersionExistsInPackaging = function (taskName, taskVersion) {
```

Comment:
this will land in a future pr, correct?

Reply:
Correct, will move it. This was mostly for experimenting.